modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/DocumentMetadataSetResourceTest.java

```diff
@@ -6,15 +6,233 @@
 package com.liferay.headless.delivery.resource.v1_0.test;
 
 import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.document.library.kernel.model.DLFileEntryMetadata;
+import com.liferay.dynamic.data.mapping.model.DDMFormField;
+import com.liferay.dynamic.data.mapping.model.DDMFormFieldOptions;
+import com.liferay.dynamic.data.mapping.model.DDMStructure;
+import com.liferay.dynamic.data.mapping.model.LocalizedValue;
+import com.liferay.dynamic.data.mapping.test.util.DDMStructureTestUtil;
+import com.liferay.headless.delivery.client.dto.v1_0.ContentStructureField;
+import com.liferay.headless.delivery.client.dto.v1_0.DocumentMetadataSet;
+import com.liferay.headless.delivery.client.dto.v1_0.Option;
+import com.liferay.headless.delivery.dto.v1_0.util.ContentStructureUtil;
+import com.liferay.petra.function.transform.TransformUtil;
+import com.liferay.portal.kernel.model.Group;
+import com.liferay.portal.kernel.test.rule.AggregateTestRule;
+import com.liferay.portal.kernel.util.LocaleThreadLocal;
+import com.liferay.portal.kernel.util.LocaleUtil;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.test.rule.PermissionCheckerMethodTestRule;
+import com.liferay.portal.vulcan.util.GroupUtil;
+import com.liferay.portal.vulcan.util.LocalizedMapUtil;
 
-import org.junit.Ignore;
+import java.util.Locale;
+import java.util.Map;
+
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
 import org.junit.runner.RunWith;
 
 /**
  * @author Javier Gamarra
  */
-@Ignore
 @RunWith(Arquillian.class)
 public class DocumentMetadataSetResourceTest
 	extends BaseDocumentMetadataSetResourceTestCase {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
+
+	@Override
+	@Test
+	public void testGetDocumentMetadataSet() throws Exception {
+		DocumentMetadataSet postDocumentMetadataSet = _addDocumentMetadataSet(
+			testGroup);
+
+		DocumentMetadataSet getDocumentMetadataSet =
+			documentMetadataSetResource.getDocumentMetadataSet(
+				postDocumentMetadataSet.getId());
+
+		getDocumentMetadataSet.setActions(postDocumentMetadataSet.getActions());
+
+		assertEquals(postDocumentMetadataSet, getDocumentMetadataSet);
+		assertValid(getDocumentMetadataSet);
+	}
```

Comment:
You need to call the parent test before adding your logic: `super.testGetDocumentMetadataSet`. See any other resource tests for examples (e.g. DocumentResourceTest).

Reply:
Thank you @adolfopa, I will add that.

Do you have any ideas on what may be causing all of the GraphQL tests to fail? I am at a loss on those tests and why they are failing.

Reply:
Can you reproduce it locally? I so, you should be able to debug and see why it is returning a null response instead of the expected JSON object.

Reply:
Yes, I can reproduce it locally. I have been trying to debug it but I have not been able to determine what is causing the issues which is why I sent the PR in hopes that someone else may have seen this kind of issue before.